components/app.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import LocationDetails from './location-details';
+import ForecastSummaries from './forecast-summaries';
+import ForecastDetails from './forecast-detils';
+import SearchForm from './search-form';
+
+import '../src/styles/app.scss';
+
+const URL = 'https://mcr-codes-weather.herokuapp.com/forecast?city=';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedDate: 0,
+      forecasts: [],
+      location: {
+        city: 'manchester',
+        country: '',
+      },
+    };
+  }
+
```

Comment:
Purely a stylistic issue, but by convention your component lifecycle methods should be at the top, followed by helpers and event handlers, finally your component render method should be at the bottom.

Reply:
Also: `goFetchNow(city)` isn't the most descriptive of method names, what is it fetching? Something like `fetchWeatherData(city)` gives other devs clues about what the purpose of the method is.